src/server/game/Entities/Creature/GossipDef.h

```diff
@@ -237,6 +234,21 @@ class TC_GAME_API QuestMenu
         QuestMenuItemList _questMenuItems;
 };
 
+class SelectionData
+{
+    public:
+        SelectionData() { Reset(); }
+
+        void Reset()
```

Comment:
you need to add a new packet class, the packet has one guid iirc, but maybe we can declare implementing that packet of of score of this pr